web_clickjack_protection/readme/ROADMAP.rst

```diff
@@ -0,0 +1,2 @@
+* Protect from Clickjacking.
+* Provide additional layer of security for Odoo user & clients.
```

Comment:
Why is this in ROADMAP? It's what the module is already offering. IMO it can be removed

Reply:
Fixed